samples/nrf9160/nrf_cloud_rest_cell_location/src/main.c

```diff
@@ -418,6 +418,12 @@ int init(void)
 		return err;
 	}
 
+	err = nrf_modem_lib_init(NORMAL_MODE);
+	if (err) {
+		printk("Modem library initialization failed, error: %d\n", err);
```

Comment:
```suggestion
		LOG_ERR("Modem library initialization failed, error: %d\n", err);
```

Reply:
@jayteemo -- there's an extra \n in your LOG_ERR